test/middlewares/error_handling.js

```diff
@@ -22,49 +22,57 @@ describe('Error handling middleware', () => {
   let request;
   let response;
   let next;
+  let mockLogger;
 
   beforeEach(async () => {
+    mockLogger = {
+      info: sinon.stub(),
+      error: sinon.stub()
+    };
     request = httpMocks.createRequest({});
     response = httpMocks.createResponse();
     next = sinon.spy();
   });
 
 
   it('should return 400 if ValidationError', async () => {
-    errorHandling(new ValidationError(), request, response, next);
+    errorHandling(mockLogger)(new ValidationError(), request, response, next);
```

Comment:
instead of instantiating a middleware each time, you could do it once in the beforeEach